cmd/subnetcmd/deploy.go

```diff
@@ -566,3 +606,185 @@ func getKeychain(
 	}
 	return sf.KeyChain(), nil
 }
+
+func checkSubnetAuthKeys(subnetAuthKeys []string, controlKeys []string, threshold uint32) error {
+	// get keys for blockchain creation
+	if subnetAuthKeys != nil && len(subnetAuthKeys) != int(threshold) {
+		return fmt.Errorf("number of given chain creation keys differs from the threshold")
+	}
+	for _, subnetAuthKey := range subnetAuthKeys {
+		found := false
+		for _, controlKey := range controlKeys {
+			if subnetAuthKey == controlKey {
+				found = true
+			}
+		}
+		if !found {
+			return fmt.Errorf("subnet auth key %s does not belong to control keys", subnetAuthKey)
+		}
+	}
+	return nil
+}
+
+func getSubnetAuthKeys(controlKeys []string, threshold uint32) ([]string, error) {
+	var subnetAuthKeys []string
+	if len(controlKeys) == int(threshold) {
+		subnetAuthKeys = controlKeys
+	} else {
+		subnetAuthKeys = []string{}
+		filteredControlKeys := controlKeys
+		for len(subnetAuthKeys) != int(threshold) {
+			subnetAuthKey, err := app.Prompt.CaptureList(
+				"Choose a chain creation key",
+				filteredControlKeys,
+			)
+			if err != nil {
+				return nil, err
+			}
+			subnetAuthKeys = append(subnetAuthKeys, subnetAuthKey)
+			filteredControlKeysTmp := []string{}
+			for _, controlKey := range filteredControlKeys {
+				if controlKey != subnetAuthKey {
+					filteredControlKeysTmp = append(filteredControlKeysTmp, controlKey)
+				}
+			}
+			filteredControlKeys = filteredControlKeysTmp
+		}
+	}
+	return subnetAuthKeys, nil
+}
+
+func saveTxToDisk(tx *txs.Tx, outputTxPath string) error {
+	ux.Logger.PrintToUser("")
+	ux.Logger.PrintToUser("Partial signing was done on blockchain tx. Saving tx to disk to enable remaining signing.")
+	// get path
+	if outputTxPath == "" {
+		ux.Logger.PrintToUser("")
+		var err error
+		outputTxPath, err = app.Prompt.CaptureString("Path to export partially signed tx to")
+		if err != nil {
+			return err
+		}
+	}
+	// Serialize the signed tx
+	txBytes, err := txs.Codec.Marshal(txs.Version, tx)
+	if err != nil {
+		return fmt.Errorf("couldn't marshal signed tx: %w", err)
+	}
+
+	// Get the encoded (in hex + checksum) signed tx
+	txStr, err := formatting.Encode(formatting.Hex, txBytes)
+	if err != nil {
+		return fmt.Errorf("couldn't encode signed tx: %w", err)
+	}
+	// save
+	if _, err := os.Stat(outputTxPath); err == nil {
```

Comment:
Do we want to add a `--force` flag for overwrites? It might be overkill, but maybe. Open to thoughts.

Reply:
I think it may be overkill. All the current flags are somewhat needed, but probably we should avoid adding more avoidable flags. Anyway if you insist will add it.